### ISSUE TITLE
Step 6: rewrite each splitted netcdf file into a torch tensor file

### ISSUE THREAD

**Issue by chengcli** [MIPlanets/planets#47]
After step 5 is finished, parse each file and rewrite the content into a torch tensor for dynamic simulation.

The script to save tensors to a file (using jit) looks like:

```python
def save_tensors(tensor_map: dict[str, torch.Tensor], filename: str):
    class TensorModule(torch.nn.Module):
        def __init__(self, tensors):
            super().__init__()
            for name, tensor in tensors.items():
                self.register_buffer(name, tensor)

    module = TensorModule(tensor_map)
    scripted = torch.jit.script(module)  # Needed for LibTorch compatibility
    scripted.save(filename)
```

Note that the variables after step5 have the following ordering (time, x1, x2, x3)
There are two steps involved,
1) Sum the following variables into one quantity
ciwc + clwc -> q2
cswc + crwc -> q3
2) Aggregate the following variables into a new tensor named 'hydro_w' of dimension (time, nvar, x3, x2, x1) (reordering of axis needed)
(rho, w, v, u, p, q, q2, q3) -> hydro_w
In this case, nvar = 8 and the order of it is exactly as above. (rho, w, v, u, p, q) are available in the original splited file, q2 and a3 are computed by the previous step

Save the new hydro_w tensor to a ".pt" file with the same basename as the ".nc" file
